docs_src/examples/Ch-OSImageWithEdgeX.md

```diff
@@ -182,42 +157,31 @@ cat model.signed.yaml
 
-> **Note**  
-> If you plan to use an emulator to install and run Ubuntu Core from the resulting image, it is a good idea to allocate additional writable storage. This necessary if you want to install additional snaps interactively or upgrade existing ones on the emulator.
->
-> The default size of the `ubuntu-data` partition is `1G` as defined in the gadget snap. When installing on actual hardware, this partition extends automatically to take the whole remaining space on the disk volume. However, when using QEMU, the partition will have the exact same size because the image size is calculated based on the defined partition structure. The 1GB `ubuntu-data` partition will be 90% full after first boot. You can configure the image to be larger so that the installer expands the partition automatically as with a large disk volume. 
->
-> To extend the image size, use the `--image-size` flag in the following command. For example, to add 500MB extra (the original image is around 3.5GB), set `--image-size=4G`.
+    The default size of the `ubuntu-data` partition is `1G` as defined in the gadget snap. When installing on actual hardware, this partition extends automatically to take the whole remaining space on the disk volume. However, when using QEMU, the partition will have the exact same size because the image size is calculated based on the defined partition structure. The 1GB `ubuntu-data` partition will be mostly full after first boot. You can configure the image to be larger so that the installer expands the partition automatically as with a large disk volume. 
+
+    To extend the image size, use the `--image-size` flag in the following command. For example, to add 500MB extra (the original image is around 3.5GB), set `--image-size=4G`.
 
 ```bash title="🖥 Desktop"
-$ ubuntu-image snap model.signed.yaml --validation=enforce --snap pc-amd64-gadget/pc_20-0.4_amd64.snap 
+$ ubuntu-image snap model.signed.yaml --validation=enforce
 Fetching snapd
 Fetching pc-kernel
-Fetching core20
 Fetching core22
+Fetching pc
 Fetching edgexfoundry
 Fetching edgex-device-virtual
-WARNING: "pc" installed from local snaps disconnected from a store cannot be refreshed subsequently!
-Copying "pc-amd64-gadget/pc_20-0.4_amd64.snap" (pc)
 
-# check the image file
+# check the created image file
 $ file pc.img
 pc.img: DOS/MBR boot sector, extended partition table (last)
 ```
 
-The warning is because we side-loaded the gadget for demonstration purposes. In production settings, a custom gadget would need to be uploaded to the [IoT App Store](https://ubuntu.com/internet-of-things/appstore) to also receive updates.
-
-!!! note
-    You need to repeat the build every time you change and sign the **model** or rebuild the **gadget**.
-
```

Comment:
There is no longer a custom gadget in this step and I thought the need to rebuild the image after changing the model assertion was obvious since the signed model assertion is the input to the build command.